src/util/is-root-node.js

```diff
@@ -0,0 +1,3 @@
+export default function(node) {
+  return node.tagName === '_SHADOW_ROOT_';
```

Comment:
We do but it's configurable via the `attachShadow()` options so we'd have to pass that around. Just created https://github.com/skatejs/named-slots/issues/92. I didn't want to do that here because I wanted to keep the PR's separate.